test/example/echo/echo.yaml

```diff
@@ -2,34 +2,54 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: echoheaders
+  name: service1
   labels:
-    app: echoheaders
+    app: service1
 spec:
   type: NodePort
   ports:
   - port: 80
     targetPort: 8080
     name: http
   selector:
-    app: echoheaders
+    app: service1
 ---
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: echoheaders
+  name: service1
 spec:
   replicas: 3
   template:
     metadata:
       labels:
-        app: echoheaders
+        app: service1
+      annotations:
+        pod.beta.kubernetes.io/init-containers: '[
+          {
+            "name": "iptables",
+            "image": "gcr.io/istio-test/init:test",
+            "imagePullPolicy": "Always",
+            "securityContext": {
+              "privileged": true
+            },
+            "command": ["/bin/bash", "-c", "sudo iptables -t nat -A PREROUTING -p tcp --dport 8080 -j REDIRECT --to-port 9999"]
+          }
+        ]'
     spec:
```

Comment:
init containers require 1.1. TPR is a biggest constraint. They'll work properly in >1.5 and might need more manual plumbing in earlier versions.